modules/ROOT/pages/deployment/services/s-list/antivirus.adoc

```diff
@@ -0,0 +1,53 @@
+= Antivirus Service Configuration
+:toc: right
+:description: The Infinite Scale antivirus service is responsible for scanning files for viruses.
+
+:ext_name: antivirus
+
+// remember to REMOVE the corresponding tab if a new ocis release will be published
+
+:no_second_tab: true
+:no_third_tab: true
+
+== Introduction
+
+{description} 
+
+== Configuration
+
+=== Antivirus Scanner Type
+
+The antivirus service currently supports https://tools.ietf.org/html/rfc3507[ICAP] and http://www.clamav.net/index.html[clamAV] as antivirus scanners. The `ANTIVIRUS_SCANNER_TYPE` environment variable is used to select the scanner. The detailed configuration for each scanner heavily depends on the scanner type selected. See the environment variables for more details.
+
+  -   For `icap`, only scanners using the `X-Infection-Found` header are currently supported.
+  -   For `clamav` only local sockets can currently be configured.
+
+=== Maximum Scan size
+
+Several factors can make it necessary to limit the maximum file size the antivirus service will use for scanning. Use the `ANTIVIRUS_MAX_SCAN_SIZE` environment variable to scan only a given amount of bytes. Obviously it is recommended to scan the whole file, but several factors like scanner type and version, bandwith, performance issues, etc. might make a limit necessary.
+
+=== Infected File Handling
+
+The antivirus service allows three different ways of handling infected files. Those can be set via the `ANTIVIRUS_INFECTED_FILE_HANDLING` environment variable:
+
+  -   *delete*: (default): Infected files will be deleted immediately, further post-processing is cancelled.
+  -   *abort*:  (advanced option): Infected files will be kept, further post-processing is cancelled. Files can be manually retrieved and inspected by an admin. To identify the file for further investigation, the antivirus service logs the abort/infected state including the file ID. The file is located in the `storage/users/uploads` folder of the ocis data directory and persists until it is manually deleted by the admin via the xref:{s-path}/storage-users.adoc#manage-unfinished-uploads[Manage Unfinished Uploads] command.
+  -   *continue*:  (obviously not recommended): Infected files will be marked via metadata as infected but post-processing continues normally. Note: Infected Files are moved to their final destination and therefore not prevented from download which includes the risk of spreading viruses.
+
+In all cases, a log entry is added declaring the infection and handling method and a notification via the `userlog` service sent.
+
+=== Scanner Inaccessability
+
+In case a scanner is not accessible by the antivirus service like a network outage, service outage or hardware outage, the antivirus service uses the `abort` case for further processing, independent of the actual setting made. In any case, an error is logged noting the inaccessability of the scanner used.
```

Comment:
```suggestion
In case a scanner is not accessible by the antivirus service like a network outage, service outage or hardware outage, the antivirus service uses the `abort` case for further processing, independent of the actual setting made. In any case, an error is logged noting the inaccessibility of the scanner used.
```